NanoProd/crab/overseer_cfg.yaml

```diff
@@ -14,6 +14,7 @@ filesToTransfer:
   - RunKit/skim_tree.py
   - RunKit/sh_tools.py
   - NanoProd/data/skim.yaml
+  - NanoProd/data/PNetCodeForCmsRun.py
```

Comment:
should be part of customise. E.g. in line 3 instead of `customize.customize` it will be `customize.customize_pnet`

Reply:
Did you mean as I've implemented it now? This didn't work out of the box though, I'm not sure what went wrong, but I had to add [this](https://github.com/cms-tau-pog/NanoProd/pull/10/files#diff-d6acc14b95ee2807fa47f0e2d8cc987f6e8446f3ae394a053ed53b940440d154R207-R217) to make it work.

However, it does seems to work now, both locally and using the dryrun crab submission, and the outputs are also the same (comparing the contents of the branches per event) as they were previously.

Reply:
yes, it is what I had in mind. I need to think why you need to apply the workaround, but probably it is ok, since it works and produces compatible results.